packages/adapter/addon/index.ts

```diff
@@ -66,6 +66,8 @@ type SnapshotRecordArray = import('@ember-data/store/-private/system/snapshot-re
   @extends EmberObject
 */
 export default class Adapter extends EmberObject implements MinimumAdapterInterface {
+  declare _coalesceFindRequests: boolean;
```

Comment:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#the-usedefineforclassfields-flag-and-the-declare-property-modifier

Reply:
WAT. I missed this discussion and I am sad. https://github.com/tc39/proposal-class-fields/issues/151#issuecomment-431597270

Reply:
I think a better alternative here may be to do this within the constructor.